modules/ximgproc/include/opencv2/ximgproc/color_match.hpp

```diff
@@ -0,0 +1,68 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef __OPENCV_COLOR_MATCH_HPP__
+#define __OPENCV_COLOR_MATCH_HPP__
+#ifdef __cplusplus
```

Comment:
`#ifdef __cplusplus` is not needed in .hpp files.

Reply:
ok

Reply:
OK